master/buildbot/schedulers/trysched.py

```diff
@@ -117,6 +120,9 @@ def activate(self):
 
     @defer.inlineCallbacks
     def deactivate(self):
+        if not self.enabled:
```

Comment:
How should I go about testing each version of twisted/python? Is there an example of this somewhere?